src/shared/components/mutationTable/column/styles.module.scss

```diff
@@ -0,0 +1,9 @@
+.cosmic-count {
```

Comment:
since `mutation-count` and `cosmic-count` styles are the same, maybe unite them as `numeric-column-value` class or something like that?